### PR TITLE
 CHE-6844: Fix Git markers on maven multi-module projects

### PR DESCRIPTION
**Pull request by vinokurig**
### What does this PR do?
Fix Git markers on maven multi-module projects

### What issues does this PR fix or reference?
#6844 

Fix Git markers on maven multi-module projects

#### Release Notes

N/A

#### Docs PR

N/A

### CODE REVIEW

**Comment by vinokurig** [comment]
ci-test

File: ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionEditorPresenter.java [hunk 1]
```diff
@@ -1233,6 +1238,16 @@ public void onContextMenu(ContextMenuEvent event) {
                 }));
   }
 
+  private Container getLastParent(Container container) {
```
Comment (tolusha) [inline ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionEditorPresenter.java]: Get rid of recursion pls
```
Container root  = getRoot(project);
...
 private Container getRoot(Container container) {
   Container root = container;
  while (root.getParent() != null) {
    root = root.getParent();
  } 

  return root;
}
```

**Review by tolusha — APPROVED** [review]

**Review by vparfonov — APPROVED** [review]

**Comment by vinokurig** [comment]
All failed tests passed successfully after rerunning them locally

File: ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionEditorPresenter.java (hunk 1 above)
Comment (vinokurig) [reply ide/che-core-orion-editor/src/main/java/org/eclipse/che/ide/editor/orion/client/OrionEditorPresenter.java]: done

**Pull request merged**